### DISCUSSION TITLE
Can I transfer zon2nix to nix-community

### DISCUSSION THREAD

**Discussion by figsoda** [nix-community/infra#802] [Q&A] [answered]
Hi! I would like to transfer [zon2nix](https://github.com/figsoda/zon2nix) to nix-community

**Comment by zowoq** [comment, marked as answer]
The repo transfer permissions are enabled so you should be able to move it, let me know if need anything else.

**Reply by figsoda** [reply]
Transferred! Can I get the admin privileges?

**Reply by zowoq** [reply]
Changed permissions to admin.
